features/tests/test_api.py

```diff
@@ -341,6 +343,34 @@ def test_feature_opening_hours(snapshot, api_client):
     snapshot.assert_match(executed)
 
 
+def test_price_list(snapshot, api_client):
```

Comment:
should this be renamed to `test_price_tags`? 😅

Reply:
as it's an API, not a model test, it does actually test the price_list. 
`properties {
                  details {
                    priceList {
                      item
                      price
                      unit
                    }
                  }`

Reply:
ah, right! 👍